crates/client/rpc/src/versions/admin/v0_1_0/api.rs

```diff
@@ -11,6 +11,6 @@ pub trait MadaraWriteRpcApi {
     #[method(name = "addDeclareV0Transaction")]
     async fn add_declare_v0_transaction(
         &self,
-        declare_transaction_v0: BroadcastedDeclareTransactionV0,
-    ) -> RpcResult<DeclareTransactionResult>;
+        _declare_v0_transaction: BroadcastedDeclareTransactionV0,
```

Comment:
This field is still in use!

```suggestion
        declare_v0_transaction: BroadcastedDeclareTransactionV0,
```

Reply:
fixed